frontend/assets/stylesheets/components/_hero-banner.scss

```diff
@@ -8,13 +8,18 @@
  */
 .hero-banner {
     color: $white;
+    height: 60vh;
     background-position: top center;
     background-repeat: repeat;
     background-attachment: fixed;
-    height: 60vh;
+    background-size: 100%;
+
+    @include mq(phablet) {
```

Comment:
I use the term to be consistent in language with frontend, but it always reminds me of this wonderful tweet https://twitter.com/kissane/status/240999346865336320